ansible/roles/execut-api/templates/.env.j2

```
@@ -0,0 +1,3 @@
+# {{ ansible_managed }}
+DATABASE_URL={{ database_url }}
+JWT_SECRET={{ jwt_secret }}
```

secrets have to be prepended with `secret_`, and we scope them by application:
```suggestion
JWT_SECRET={{ secret_execute_jwt_token }}
```